pkgs/shells/zsh/zplugin/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+stdenvNoCC.mkDerivation rec {
+  pname = "zplugin";
+  version = "2.3";
+  src = fetchFromGitHub {
+    owner = "zdharma";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0qqv5p19s8jb06d6h55dm4acji9x2rpxb2ni3h7fb0q43iz6y85w";
+  };
+  # adapted from https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=zsh-zplugin-git
+  dontBuild = true;
+  installPhase = ''
+    outdir="$out/share/$pname"
+
+    cd "$src"
+
+    # Zplugin's source files
+    install -dm0755 "$_plugindir"
+    install -m0644 zplugin{,-side,-install,-autoload}.zsh "$outdir"
+    install -m0755 git-process-output.zsh "$outdir"
+    install -m0744 _zplugin "$out/share/zsh/site-functions"
+
+    # Zplugin-module files
+    find zmodules/ -type d -exec install -dm 755 "{}" "$outdir/{}" \;
+    find zmodules/ -type f -exec install -m 744 "{}" "$outdir/{}" \;
+
+  '';
+  #TODO:doc output
+  
+  meta = with lib; {
+    license = licenses.mit;
+    homepage = "https://github.com/zdharma/zplugin";
+    description = "Flexible zsh plugin manager";
+    license = licenses.mit;
```

please add yourself as a maintainer :)
```suggestion
    license = licenses.mit;
    maintainers = with maintainers; [ pasqui23 ];
```

---

Wait I'm not listed among the mantainers,can I add myself?